test/api.mocha.coffee

```diff
@@ -350,6 +356,11 @@ describe "API", ->
 
               # Send them invitations
               (_party, cb) ->
+                 # @TODO - _party should be array of newly
```

Comment:
[This line is why](https://github.com/HabitRPG/habitrpg/blob/a8cb0fd4e9785f257ca5edbc9c62dba7577d5ca2/test/api.mocha.coffee#L21)